assets/queries/openAPI/2.0/schema_with_additional_properties_set_as_boolean/query.rego

```diff
@@ -0,0 +1,19 @@
+package Cx
+
+import data.generic.openapi as openapi_lib
+
+CxPolicy[result] {
+	doc := input.document[i]
+	openapi_lib.check_openapi(doc) == "2.0"
+
+	[path, value] := walk(doc)
+	is_boolean(value.schema.additionalProperties)
```

Comment:
I think here can be just `value.additionalProperties`, no? Since there is `additionalProperty` only for schema objects... Also, if we checks `value.schema.additionalProperties`, we won't check `definitions` object, since it does not have a 'schema' property...

```suggestion
	is_boolean(value.additionalProperties)
```

Reply:
I thought it does not make sense to check `definitions object` since it is unusual for a Definitions Object to refer to another Definitions Object. What do you think?

Reply:
In the first instance, I thought that does not make sense to check `definitions object`, since it is unusual for a Definitions Object to refer to another Definitions Object. However, on second thought, it makes sense since `additionalProperty` could be something else than `$ref`:
```
additionalProperties:
  type: string
```